armi/physics/fuelPerformance/parameters.py

```diff
@@ -27,6 +27,13 @@ def _getFuelPerformanceBlockParams():
     pDefs = parameters.ParameterDefinitionCollection()
     with pDefs.createBuilder(default=0.0, location=ParamLocation.AVERAGE) as pb:
 
+        pb.defParam(
+            "fuelCladLocked",
+            units=bool,
```

Comment:
Units is not a boolean, right?

Reply:
Yeah, the boolean isn't the units. I was thinking that maybe we could use it to enfore type, but now that you comment on it, that doesn't make sense. I'll pop it out.

Reply:
the type gets inferred from the default

Reply:
Addressed https://github.com/terrapower/armi/pull/1038/commits/67dee5406cd9fa7da82cfc1287a7b636500956d9